src/log.cpp

```diff
@@ -338,7 +339,65 @@ void FileLogOutput::setFile(const std::string &filename, s64 file_size_max)
 		"-------------\n" << std::endl;
 }
 
+void StreamLogOutput::logRaw(LogLevel lev, const std::string &line)
+{
+	bool colored_message = (Logger::color_mode == LOG_COLOR_ALWAYS) ||
+		(Logger::color_mode == LOG_COLOR_AUTO && is_tty);
+	if (colored_message) {
```

Comment:
No functional changes. Added braces here according to the code style guidelines.